ops/terraform/services/server/server-load/main.tf

```diff
@@ -1,23 +1,22 @@
-locals {
-  default_tags = {
-    Environment    = local.seed_env
-    Layer          = local.layer
-    Name           = "bfd-${local.env}-${local.service}"
-    application    = "bfd"
-    business       = "oeda"
-    role           = local.service
-    stack          = local.env
-    Terraform      = true
-    tf_module_root = "ops/terraform/services/server/server-load"
+module "terraservice" {
+  source = "../../_modules/bfd-terraservice"
+
+  environment_name     = terraform.workspace
+  relative_module_root = "ops/terraform/services/server/server-load"
+  additional_tags = {
+    Layer = local.layer
+    Name  = "bfd-${local.env}-${local.service}"
+    role  = local.service
   }
+}
 
+locals {
   account_id             = data.aws_caller_identity.current.account_id
   availability_zone_name = var.create_locust_instance ? data.aws_availability_zones.this.names[random_integer.this[0].result] : ""
 
-  env              = terraform.workspace
-  established_envs = ["test", "prod-sbx", "prod"]
-  seed_env         = one([for x in local.established_envs : x if can(regex("${x}$$", local.env))])
-  is_ephemeral_env = !(contains(local.established_envs, local.env))
+  default_tags = module.terraservice.default_tags
+  env          = module.terraservice.env
+  seed_env     = module.terraservice.seed_env
 
```

Comment:
Was the ommission of is_ephemeral_environment deliberate?

Reply:
That's intentional. This was mostly copy/pasta or _cargoculting_.